sdk/core/Azure.Core/src/Shared/AzureKeyCredentialPolicy.cs

```diff
@@ -7,6 +7,9 @@
 
 namespace Azure.Core
 {
+#if NET6_0_OR_GREATER
+    [System.Diagnostics.CodeAnalysis.DynamicallyAccessedMembers(System.Diagnostics.CodeAnalysis.DynamicallyAccessedMemberTypes.PublicMethods)]
```

Comment:
All these `DynamicallyAccessedMembers` on the type are very likely not doing what you think they're doing. Where is the place this is used - is in - where is the reflection which requires public methods on these types?

Reply:
All of these are resolving the same warning - https://github.com/Azure/azure-sdk-for-net/pull/37734/files#r1283677872